apps/src/applab/ai.js

```diff
@@ -52,6 +53,7 @@ function generateCodeDesignElements(modelId, modelData) {
   var predictButtonId = alphaNumModelName + '_predict';
   designMode.updateProperty(predictButton, 'id', predictButtonId);
   var predictOnClick = `onEvent("${predictButtonId}", "click", function() {
+    ${inputFields.toString()}
```

Comment:
Does the "text only" code generated look okay for multiple fields?

Reply:
Good call; it was one long, comma-separated line. Updated ✔️